code/modules/antagonists/darkspawn/darkspawn_abilities/_divulge.dm

```diff
@@ -0,0 +1,79 @@
+//A channeled ability that turns the darkspawn into their main form.
+/datum/action/innate/darkspawn/divulge
+	name = "Divulge"
+	id = "divulge"
+	desc = "Sheds your human disguise. This is obvious and so should be done in a secluded area. You cannot reverse this."
+	button_icon_state = "divulge"
+	check_flags = AB_CHECK_STUN | AB_CHECK_CONSCIOUS | AB_CHECK_LYING
+	blacklisted = TRUE
+
+/datum/action/innate/darkspawn/divulge/Activate()
+	set waitfor = FALSE
+	var/mob/living/carbon/human/user = usr
+	var/turf/spot = get_turf(user)
+	if(spot.get_lumcount() > DARKSPAWN_DIM_LIGHT)
+		to_chat(user, "<span class='warning'>You are only able to divulge in darkness!</span>")
+		return
+	if(alert(user, "You are ready to divulge. Are you sure?", name, "Yes", "No") == "No")
+		return
+	in_use = TRUE
+	user.visible_message("<b>[user]</b> flaps their wings.", "<span class='velvet'>You begin creating a psychic barrier around yourself...</span>")
```

Comment:
hmm...

Reply:
since they're still disguised, and you can type `*flap` to make an emote of the same text, I thought it would be an amusing touch, since because you are not yet an evil alien from hell, you probably just look like you're flailing around